drivers/gles2/shader_compiler_gles2.cpp

```diff
@@ -755,11 +755,11 @@ String ShaderCompilerGLES2::_dump_node_code(SL::Node *p_node, int p_level, Gener
 
 				case SL::OP_MOD: {
 
-					code += "mod(float(";
+					code += "int(floor(0.5+mod(float(";
```

Comment:
I think the `floor()` is unnecessary, `int()` will already truncate. Please also add spaces around `+` to use the same style as for our C++ code (even though we can't run `clang-format` on this generated code).

Reply:
Right, makes sense.